### ISSUE TITLE
Match rewards and currency recommendations

### ISSUE THREAD

**Issue by WongKit** [sstokic-tgm/JFTSE#58]
I think it is a good time to collect some of my thoughts about the exp and currency handling. Please see the following points as a recommendation from a former FT player:

**Match rewards**

The "original" match rewards sucked, so I won't explain them here.

The match rewards should be encourage players to play and not to win or destroy the opponent. So here is my idea:

- Create a base reward which is based on the match play time. Limit it between 1:30 minutes and 15:00 Minutes. Playing for 15 Minutes or more should give a base reward of 100 exp. Playing for 1:30 minutes should give you around 30 exp. As a formular, it should be [**30 + (matchseconds - 90) x 0.0864**].
 If you play for less, only reward 1 exp to prevent exploiting.
Battle mode should have a lower minimum limit (e.g. 1 minute) as those matches can be over faster.
- Add a percentage bonus of 20% to the base reward for the winning team
- Add a percentage bonus of 5% to the base reward for each of the result bonuses (Guard Break, Perfect, Double,...)
- Add a percentage bonus of [Housing Bonus] to the base reward
- Add a percentage bonus of 10% and 5% to the base reward for best and 2nd best player of a match. The player performance should be determined by their ball hit count
- Giving a percentage bonus for a weaker team (based on total str/sta/dex/wil and winning statistics) who wins could be also a good idea, but I don't know if this too much of a calculation/hassle
- The gold reward should be based on the total exp reward. Factor x1.0 is fine for the moment.
- The original server had some form of protection that reduced the rewards to 1 if you played against the same person in single mode multiple times (4+) in a row. This could also prevent farming.

**Guardian rewards**

- The base rewards should be based on the enemy type, count and level. It should range from 10 exp to 150 *per enemy*.
- Add a percentage bonus of 5% to the base reward for each of the result bonuses (Guard Break, Perfect,...)
- Beating the guardian should also give you the item reward roulette with guardian specific items.

**AP currency**

I like the idea of still keeping this currency and make it obtainable by different ways.
- Reward AP currency for beating repeatable tasks in the missions category
- Reward AP currency once a day for logging in
- Reward AP currency for special events like tournaments
- Do timed events where you can get AP currency on normal matches based on exp rewards. Maybe add a multiplier to it.
- *Rewarding lots of AP currency for NOT creating a battlemon room* :P
- AP Gacha equipment drop rates should be increased as this is no money milking anymore

As you might noticed, I have not put any numbers in the AP currency section. It would take more time to analyse the value of the premium items and their cost to give a more detailed recommendation.

**Couple points**

I am not sure, if there are that useful as there are not many items you can buy. If you want to utilise them, please rename them to **Team points** and allow couple between any character class.

I am crossposting my recommendations to discord as well for further discussion.

**Comment by moynmoyn** [comment]
whats the point of this. theres not enough active users just do everyone level 50 for now and look how it goes.

**Comment by Cive1212** [comment]
Hey so i have few ideas about exp and currencies in the game. Overall i like what you did there and i like the idea of keeping AP as a 2nd currency in the game. Here are some additional thoughts:

I had a discusion with few players on discord and a lot of them said the same thing "i don't have as much time to play as i did back then and lvling up and getting gold should be way faster now!".
as much as im aware most of the old FT players are 20+ now and doesen't have as much times as they used to a multiplayer game should always reward those who invest more time and at the same time don't make people with little time feel mad about no progress. I think it would be a nice idea to implement exp stages for example:
lvl 1 to lvl 20 you would get 4x more exp 
lvl 20 to 30 3x more
lvl 30 to 40 2x more exp
lvl 40+ you would get normal exp (x1 stage)

this will make people with less time get enough lvl to compete with others (hit lvl 40 fast) and at the same time people with more time won't achieve everything in 2 weeks cuz from lvl 40+ progress will slow down alot. This might also be usefull in the future if someday you will decide to add more lvls for example set max lvl to 70 then you might put an exp stage from 60 to 70 to 0.5 for the most dedicated players!

About the AP currency. I think it would be nice to implement ap boxes as a rare drop from guardian mode. The amount of AP in box would depend of a map you play for example small ap box could drop from the "easy" maps (up to aerials) medium ap box would drop from medium difficulty maps (forest,arena etc) big ap boxes would drop from hardest maps (atlantis, devaberg) and super ap boxes would drop from boss fights. Obviously the better the box the more AP you get for example small box 10-18 big box 30-45 etc.

I think the amount of AP you get from either boxes or some tasks/daily log in should be small to not get everything fast. Also it should be more beneficial to farm harder maps than to grind the easiest one fast to get tons of small boxes! 

Another idea is to delete some gachas from shop and put them as a very rare rewards from guardian mode!(only hardest maps/boss fights). This will motivate people even more to get strong, farm these hard maps to get items they can't just buy in shop! 

Hope you like my ideas ^-^

**Comment by WongKit** [comment]
I agree with you, that leveling should be faster than in the EU version. They had very low exp rewards to push AP sets with low level requirements and guardian mode with AP quickslot, AP booster and AP time bonuses.

The different exp rewards per level don't make much sense to me. I mean, you want people to level up more quickly on lower levels and slower on higher ones. _This is already the case_, because higher levels require exponentially more exp to level up ;)

AP gacha/items as additional guardian drops is a good idea, It should not be forced, though.

**Comment by G1i2o3** [comment]
I agree with you guys.
Old FT version was a "pay-to-win" game. You had to spend money to get AP ecc...
Gacha system needs to be re-make completely, and in my opinion AP should be gain with daily task system, match rewards or events.
Lot of old players know how to hack a game, so they should be discouraged if game won't have old pay-to-win system, but it will only based on just "play for fun" system.
